config/resources/domain.json

```diff
@@ -299,14 +299,22 @@
           "cardinality": "many"
         },
         "changed-by": {
-          "predicate": "org:changedBy",
+          "predicate": "org:originalOrganization",
           "target": "change-events",
-          "cardinality": "many"
+          "cardinality": "many",
+          "inverse": true
         },
         "resulted-from": {
-          "predicate": "org:resultedFrom",
+          "predicate": "org:resultingOrganization",
           "target": "change-events",
-          "cardinality": "many"
+          "cardinality": "many",
+          "inverse": true
+        },
+        "changes": {
```

Comment:
probably not the most clear relationship name. Any suggestions?

Reply:
Why not changeEvents?

Reply:
It is linked to the `ChangeEventResults` so `changeEventResults` would be more correct, but that seemed to long / specific so I simplified it to `changes`. But I can go for that one if you prefer.

Reply:
I would go for clarity, i.e. `changeEventResults`